packages/ui/src/council/components/election/CandidateVote/CandidateVote.tsx

```diff
@@ -86,9 +84,13 @@ export const CandidateVote = ({
           <StakeAndVotesRow>
             {myStake?.gt(BN_ZERO) && (
               <>
-                <Tooltip tooltipText={tooltipLabel} tooltipLinkURL={tooltipLinkURL}>
+                <Tooltip
+                  tooltipText="Vote for your own membership with the same account as used for council candidacy creation counts towards the progress. Please note, this will in a voting lock applied to this account, which may only be withdrawn in the end of the council period if you win the election and released immediately if your candidacy gets outvoted by others. Voting locks are non-rivalrous."
+                  tooltipLinkText="Read more"
+                  tooltipLinkURL="https://handbook.joystream.org/system/council#vote"
+                >
```

Comment:
@gyroflaw I have no preference between passing the label and url as variables or string literals, but please try to avoid introducing unnecessary changes in the future. It simplify review and prevent conflicts.